src/extension/background-script/actions/blocklist/list.ts

```diff
@@ -0,0 +1,23 @@
+import { Blocklist } from "../../../../types";
+import db from "../../db";
+
+const list = async (message: string, sender: unknown) => {
+  // TODO add filter and ordering?
```

Comment:
Is this a `TODO` for this PR or later on? If later on, maybe create an issue and link it here

Reply:
I think this was juts copy&paste. Sorting is done in the code below this. I think the comment can be removed.